ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -141,26 +137,57 @@ const payUrlSuffix = "/v3/webview/transactions/pay";
 const webViewExitPathName = "/v3/webview/logout/bye";
 const webViewOutcomeParamName = "outcome";
 
-const PaymentMethodCard = (props: {
-  paymentMethod: PaymentMethod | undefined;
-  isPaypalEnabled: boolean;
-}) => {
-  const { paymentMethod } = props;
+const getPaymentMethodInfo = (
+  paymentMethod: PaymentMethod | undefined,
+  isPaypalEnabled: boolean
+): Option<{
+  logo: JSX.Element;
+  subject: string;
+  expiration: string;
+  caption: string;
+  accessibilityLabel: string;
```

Comment:
could it be a local type to avoid a big function signature?

Reply:
Yes you're right! Fixed here: https://github.com/pagopa/io-app/pull/3840/commits/59c24aa1aa3021c67321d65ea2cfeafc559621c6#